### PR TITLE
Update for Vaxrank 0.3.0

### PR DESCRIPTION
**Pull request by ihodes**
Fixes https://github.com/hammerlab/epidisco/issues/93

### CODE REVIEW

**Comment by ihodes** [comment]
Question for you @julia326; do I need to give Vaxrank

--output-reviewed-by xx,yy,zz
--output-final-review xx

or if those are emitted, it'll work just fine?

**Comment by julia326** [comment]
@ihodes those 2 args are optional, so you should be all good - but we should default those to us and Alex probably anyway

File: src/lib/pipeline.ml [hunk 1]
```diff
@@ -45,11 +45,23 @@ let star_config =
...
+   max_mutations_in_report = 20;
+   min_mapping_quality = 1;
+   min_variant_sequence_coverage = 1;
+   min_alt_rna_reads = 3;
+   use_duplicate_reads = false;
+   drop_secondary_alignments = false;
+   mhc_epitope_lengths = [8; 9; 10; 11];
+   reviewers = [];
+   final_reviewer = "";
+   xlsx_report = true;
```
Comment (julia326) [inline src/lib/pipeline.ml]: these should actually be file paths instead of booleans, where the reports need to be saved to

**Comment by ihodes** [comment]
I don't think they make reasonable defaults, as this is a general pipeline (for anyone to use)! I may add them to the CLI later, but I think it's just as easy to sign names on the final form (which we do manually, anyway) when we review it.) https://github.com/hammerlab/epidisco/issues/106

File: src/lib/pipeline.ml (hunk 1 above)
Comment (ihodes) [reply src/lib/pipeline.ml]: They get translated to paths in the Biokepi code that uses these args (cf. https://github.com/hammerlab/biokepi/pull/388/files#diff-14f133a2f008d5a32a8cdec39363d489R168)

Comment (julia326) [reply src/lib/pipeline.ml]: ah cool, looking at the other PR now - lgtm :)

Comment (ihodes) [reply src/lib/pipeline.ml]: In general we try to push as much "path" stuff down to Biokepi to handle, so there isn't necessarily a direct mapping between the args/types you see in the configuration and the args you've provide on the command-line.

**Comment by ihodes** [comment]
@smondet this is good to merge, btw; functioning as intended in a real pipeline run.

**Pull request merged**